blocks/library/heading/test/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { query, getBlocks, unregisterBlock, setUnknownTypeHandler } from 'api';
+import { blockSettings } from '../';
+
+describe( 'heading block', () => {
+	afterEach( () => {
+		setUnknownTypeHandler( undefined );
+		getBlocks().forEach( ( block ) => {
+			unregisterBlock( block.slug );
+		} );
+	} );
+
+	it( 'should parse the block properly', () => {
+		const rawContent = '<h4 style="text-align: right;">Chicken</h4>';
+		const attrs = query.parse( rawContent, blockSettings.attributes );
```

Comment:
I don't know, My preference would go to exporting the whole object as a default export and using that in the tests (avoid test specific exports) but this means including the slug in the object.